development/jupyterlite.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Connect to Jupyterlite</title>
+    <link rel="stylesheet" type="text/css" href="index.css" />
+    <script type="text/x-thebe-config">
+      {
+        bootstrap: false,
+        mountActivateWidget: true,
+        mountStatusWidget: true,
+        requestKernel: true,
+        jupyterlite: true,
+        kernelOptions: {
+          name: "python3",
+          kernelName: "python3",
+          path: ".",
+          serverSettings: {
+            "baseUrl": "http://127.0.0.1:8888",
+            "token": "test-secret"
+          }
+        },
+      }
+    </script>
+    <script>
+      var indexURL = "https://cdn.jsdelivr.net/pyodide/v0.18.0/full/";
+      var _widgetsnbextensionWheelUrl =
+        "../py/widgetsnbextension/dist/widgetsnbextension-3.5.0-py3-none-any.whl";
+      var _nbformatWheelUrl =
+        "../py/nbformat/dist/nbformat-4.2.0-py3-none-any.whl";
+      var _ipykernelWheelUrl =
+        "../py/ipykernel/dist/ipykernel-5.5.5-py3-none-any.whl";
+      var _pyoliteWheelUrl =
+        "../py/pyolite/dist/pyolite-0.1.0a6-py3-none-any.whl";
```

Comment:
In JupyterLite these files are built from these packages: https://github.com/jupyterlite/jupyterlite/tree/main/packages/pyolite-kernel/py

And copied to the dist directory by `webpack`.

With https://github.com/jupyterlite/jupyterlite/issues/386 maybe it will be easier to reuse these assets outside of lite.